Misc fixes

Bertram25 opened the pull request:
Fixes:
- Yellow color being too discrete.
- Claimed walls border sometimes not being marked.
- Pay day being too short.
- Default views can cycle with the V key.

akien-mga commented:
Looks fine to me in game. I love the view cycling with V. Pressing it quickly for a long time can give nausea though, beware :-P

I agree with the pay day change, the previous value was a bit short since it was meant for testing.

Commit:
Readded support for the default std game view and made them cycle with the V key.

I also made it so the tilting value is slightly higher to be able to grab certain
tiles more easily.

Fixes #494

Commit:
Multiplied the time between paydays by 2.2, after a few games.

This value sounds saner IMHO.

I'm glad we're already at the stage where we are tweaking the game, btw. :)

Commit:
Made the yellow color applied on all passes.

Fixes #496
Fixes #456

hwoarangmy commented:
Nice to switch views.

> Pressing it quickly for a long time can give nausea though, beware :-P

That's so true ^^

Can be merged

Bertram25 commented:
>  Pressing it quickly for a long time can give nausea though, beware :-P

Misuse of a feature is strongly advised if it can make me with on a lan. ;P
